<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Result_credit</class>
 <widget class="QWidget" name="Result_credit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>673</width>
    <height>488</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 15pt &quot;Times New Roman&quot;;</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>611</width>
     <height>421</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Месяц</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Сумма платежа</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Платеж по процентам</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Платеж по основному долгу</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Остаток долга</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_dolg">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>440</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    qproperty-alignment: 'AlignCenter';
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_proc">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>440</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    qproperty-alignment: 'AlignCenter';
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>460</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>(Общая выплата)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>460</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>(Сумма выплаченных процентов)</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
